lib/oneview-sdk/image-streamer/client.rb

```diff
@@ -48,7 +48,7 @@ def initialize(options = {})
         if options[:api_version] && options[:api_version].to_i > @max_api_version
           logger.warn "API version #{options[:api_version]} is greater than the Image Streamer API version (#{@max_api_version})"
         end
-        @api_version = options[:api_version] || [OneviewSDK::ImageStreamer::DEFAULT_API_VERSION, @max_api_version].min
+        @api_version = options[:api_version] || @max_api_version
```

Comment:
In case of image streamer resources, we cannot assume the OneView's current version as default api version. Because sometimes current api_version of OV and i3s may differ. (for example, when OV api version is 1200, i3s api version is 1080).
Anyway, currently we don't have new i3s api versions, so we can skip for now. But try to add some comment regarding the change to be done in future.

Reply:
Sure, have reverted to old changes wrt I3s.